src/containers/ImportResources/ImportResources.js

```diff
@@ -0,0 +1,221 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+
+// import Link from 'react-router-dom';
+import {
+  ToastNotification,
+  TextInput,
+  FormLabel,
+  Button,
+  Tooltip
+} from 'carbon-components-react';
+import { connect } from 'react-redux';
+
+import '../../components/Definitions/Definitions.scss';
+import './ImportResources.scss';
+
+import { createPipelineRun } from '../../api';
+import ServiceAccountsDropdown from '../ServiceAccountsDropdown';
+import { getSelectedNamespace } from '../../reducers';
+
+export class ImportResources extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      repositoryURL: '',
+      serviceAccount: '',
+      submitSuccess: false,
+      invalidInput: false,
+      logsURL: '',
+      directory: ''
+    };
+  }
+
+  handleServiceAccount(data) {
+    this.setState({
+      serviceAccount: data.selectedItem.text
+    });
+  }
+
+  handleTextInput(event) {
+    const identifier = event.target.name;
+    const inputValue = event.target.value;
+    this.setState({
+      [identifier]: inputValue
+    });
+  }
+
+  handleSubmit(event) {
+    const { namespace } = this.props;
+    const pipelinename = 'pipeline0';
+    const gitresourcename = 'git-source';
+    const gitcommit = 'master';
+    const applydirectory = this.state.directory;
+    const repourl = this.state.repositoryURL;
+    const serviceaccount = this.state.serviceAccount;
+    const payload = {
+      pipelinename,
+      serviceaccount,
+      gitresourcename,
+      gitcommit,
+      repourl,
+      applydirectory
+    };
+
+    console.log(payload);
+
+    const promise = createPipelineRun(payload, namespace);
+    promise.then(headers => {
+      const logsURL = headers.get('Content-Location');
+      const trimmedLogsURL = logsURL.substring(logsURL.lastIndexOf('/') + 1);
```

Comment:
Is this a pipeline run name? The variable name could reflect this to make it easier to follow the code.

We'll also need to be careful when constructing client URLs as we're currently reviewing changes in the approach / structure to align more closely with the kubernetes API structure. Anywhere we hardcode assumptions about the client URLs will need to be updated if we make that change.